packages/api/src/controllers/api-token.js

```diff
@@ -66,6 +67,11 @@ app.get("/", async (req, res) => {
 
   if (!userId) {
     const query = parseFilters(fieldsMap, filters);
+    query.push(
+      req.project?.id
+        ? sql`api_token.data->>'projectId' = ${req.project.id}`
+        : sql`api_token.data->>'projectId' IS NULL OR api_token.data->>'projectId' = ''`
```

Comment:
if you want parity with the ts checks I suggeted, I think these queries could also be:
```suggestion
      sql`coalesce(api_token.data->>'projectId', '') = ${req.project?.id || ''}`
```

Reply:
OK updated to your suggestion and will test to verify it's the same behavior.